### ISSUE TITLE
Error

### ISSUE THREAD

**Issue by Pranay1004** [thelinuxchoice/inshackle#2]
Trying to follow celebgram 12331195 ...
 [!] Error

**Comment by dikshantnaik** [comment]
Same problem 😩

**Comment by dikshantnaik** [comment]
Use Tor.. You won't get issue..!

**Comment by dikshantnaik** [comment]
Use Tor.. You won't get issue..!

**Comment by Pranay1004** [comment]
How to use tor manually?

**Comment by iraizo** [comment]
same problem

**Comment by Pranay1004** [comment]
What to do?

**Comment by Erjack12345** [comment]
Tienes que borrar el cokie id e iniciar sesion nuevamente escribe bien tu clave y listo problema resuelto

**Comment by STAR-Sec** [comment]
Same problem any solutions

**Comment by bboyarskyi** [comment]
Same problem 😩

**Comment by nerababo** [comment]
to solve this you should learn bash

**Comment by KnightSec-Official** [comment]
@nerababo no.. you need to learn instagram.. ffs.... instagram blocks accounts for using too much follow-unfollow..... the sleeping time is too short.... wait 1-2 days without following and unfollowing anyone from that account... and then start the tool again....

**Comment by KnightSec-Official** [comment]
and do not use vpn

**Comment by nerababo** [comment]
> @nerababo no.. you need to learn instagram.. ffs.... instagram blocks accounts for using too much follow-unfollow..... the sleeping time is too short.... wait 1-2 days without following and unfollowing anyone from that account... and then start the tool again....and DO NOT USE VPN
> 
> SEE #9

trying to follow celeb blabla issue is caused by the specific followed acc being deactivated, so if the user knows how to use bash he can replace that acc with another or even delete it, save the file and run the tool again without issues. after reaching max follows in 24h it is mandatory to wait 24h after reaching the quota. so if u know how to, u could write a script to get the time when instagram blocked you and start the timer to count when to start again

**Comment by anshadrazakk** [comment]
i have same  problem

**Comment by anshadrazakk** [comment]
what should i do

**Comment by samuelbol** [comment]
> Use Tor.. You won't get issue..!

How do I use tor in termux.

**Comment by etherealxx** [comment]
> How
$ pkg install tor
$ tor

**Comment by VasiokD** [comment]
So, you have to switch to instagram business account, and then change in back to personal account. This will work.
